cpp/autosar/src/rules/M0-1-4/SingleUseMemberPODVariable.ql

```diff
@@ -24,5 +24,5 @@ where
   not isExcluded(v, DeadCodePackage::singleUseMemberPODVariableQuery()) and
   isSingleUseNonVolatilePODVariable(v)
 select v,
-  "Member POD variable " + v.getName() + " in " + v.getDeclaringType().getName() + " is only $@.",
-  getSingleUse(v), "used once"
+  "Member POD variable '" + v.getName() + "' in '" + v.getDeclaringType().getName() +
```

Comment:
not sure if while we are at it, we could apply this similar alert format improvement to the other queries in this rule

Reply:
We will address incorrectly formatted alerts in a different PR once we have QL4QL rule for it. For now I'll do it for queries that I'm changing anyways.